pkg/data/gen/mergedseriesgenerator.go

```diff
@@ -0,0 +1,141 @@
+package gen
```

Comment:
The only reason I ended up doing that was because the test files don't sort correctly and was driving the OCD in me mad :)

```
series.go
series_test.go
seriesgenerator.go
seriesgenerator_test.go
```

vs

```
series.go
series_generator.go
series_generator_test.go
series_test.go
```

Reply:
Fixed